src/DateTime/RegistrationExpirationHelper.php

```diff
@@ -19,7 +19,7 @@
 namespace Surfnet\StepupBundle\DateTime;
 
 use DateInterval;
-use DateTime;
+use DateTime as CoreDateTime;
```

Comment:
In this namespace there is already a DateTime, this resolves in a error if you use it like this.

Reply:
Lint issues when backslash is used, namespace should be imported